MobiledgeX/Editor/InstallMobiledgeX.cs

```diff
@@ -0,0 +1,276 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Net;
+using System;
+using System.IO;
+
+namespace MobiledgeX
+{
+  [ExecuteInEditMode]
+  public class InstallMobiledgeX : EditorWindow
+  {
+    GUIStyle headerStyle;
+    GUIStyle labelStyle;
+    GUIStyle textAreaStyle;
+    GUIStyle buttonStyle;
+    Texture2D mexLogo;
+    const string grpcPluginsUrl = "https://packages.grpc.io/archive/2019/11/6950e15882f28e43685e948a7e5227bfcef398cd-6d642d6c-a6fc-4897-a612-62b0a3c9026b/csharp/grpc_unity_package.2.26.0-dev.zip";
```

Comment:
Ok. Downloaded. Wouldn't hurt to double check the checksum against a known value given how specific this needs to be.

Reply:
I am not calling an API and passing some params, I am hardcoding the download url.
Do you think with the hardcoded URL there is any chance of not getting the correct package?

Reply:
^ sorry for not responding to this. Maybe it's fine. The URL is pretty specific anyway, and HTTPs .